packages/@aws-cdk/aws-servicecatalog/lib/product.ts

```diff
@@ -77,9 +77,9 @@ export interface CloudFormationProductProps {
 
   /**
    * The language code.
-   * @default - No accept language provided
+   * @default - No message language provided
    */
-  readonly acceptLanguage?: AcceptLanguage;
+  readonly messageLanguage?: MessageLanguage;
```

Comment:
You need to enumerate each breaking change in the PR description. The "Contributing guide" shows you how to format the message with multiple breaking changes: https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#step-4-pull-request

Reply:
oh so it needs to be the change for each API that is affected?